R/find_predictors.R

```diff
@@ -164,6 +164,10 @@ find_predictors.fixest <- function(x, flatten = FALSE, ...) {
 
   conditional <- all.vars(stats::formula(x))
   conditional <- setdiff(conditional, c(instruments, cluster, find_response(x)))
+  # Catch for interacted fixest::i(f0, i.f2)
+  if (any(grepl(", i.", x$fml))) {
```

Comment:
1. Should `fixed = TRUE` here? Else, `.` is considered as regex char, if I'm not mistaken
2. What about whitespace? Does this pattern also capture `fixest::i(f1,i.f2)` (no whitespace)?

Reply:
1. Oh, quite right! Sorry, I was rushing.
2. From my testing earlier, I believe so. I'll double check now and change if needed.